services/order.py

```diff
@@ -0,0 +1,47 @@
+from datetime import datetime
+
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Ticket, Order, MovieSession
+
+
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: str = None
+) -> None:
+    with transaction.atomic():

```

Comment:
Consider using the `@transaction.atomic` decorator for better readability instead of `with transaction.atomic:`.